lib/validate.js

```diff
@@ -124,7 +124,37 @@ const validateCategory = (
     const isSchemaProp = schemaProps.has(prop);
     const isObjectProp = objectProps.has(prop);
     if (isObjectProp && !isSchemaProp) {
-      errors.push(new ValidationError('unresolvedProperty', `${path}${prop}`));
+      if (schema[Symbol.for('other')]) {
+        const propPath = `${path}${prop}`;
+        const valid = schema[Symbol.for('other')].some(type => {
+          let kind, name;
+          if (type.domain) {
+            kind = 'domains';
+            name = type.domain;
+          } else {
+            kind = 'category';
+            name = typeof type === 'string' ? type : type.category;
+          }
+          if (
```

Comment:
```suggestion
          return !ms.validate(kind, name, instance[prop], { path: propPath, patch });
```

instead of lines 138-142.